<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>731</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>630</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>341</width>
        <height>291</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>버디버디.png</pixmap>
      </property>
     </widget>
     <widget class="QTabWidget" name="acount_tabWidget">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>300</y>
        <width>331</width>
        <height>201</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>로그인</string>
       </attribute>
       <widget class="QPushButton" name="chat_open_btn">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>130</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>들어가기</string>
        </property>
       </widget>
       <widget class="QLabel" name="login_label">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>90</y>
          <width>241</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>200</width>
          <height>48</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>PHONE</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="user_phone_lineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>PASS</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="user_pass_lineEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>회원가입</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>226</width>
          <height>96</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>휴대폰 번호</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>이름</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>비밀번호</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="phone_lineEdit">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="check_label">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="name_lineEdit"/>
           </item>
           <item>
            <widget class="QLineEdit" name="pass_lineEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="join_btn">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>150</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>회원가입</string>
        </property>
       </widget>
       <widget class="QLabel" name="join_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>241</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="check_btn">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>30</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>중복확인</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>610</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLabel" name="user_name_label_2p">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="chat_name_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="user_name_label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>채팅방 목록</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="chat_list_tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>261</width>
        <height>491</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <column>
       <property name="text">
        <string>번호</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>채팅방</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="chat_search_btn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>채팅방 조회</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="chat_name_line_edit">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>80</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="chat_add_btn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>채팅방 추가</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="chat_groupBox">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>441</width>
        <height>561</height>
       </rect>
      </property>
      <property name="title">
       <string>채팅방: </string>
      </property>
      <widget class="QScrollBar" name="signal_ScrollBar">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>470</y>
         <width>20</width>
         <height>71</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="receive_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>수신 메시지:</string>
       </property>
      </widget>
      <widget class="QLabel" name="signal_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>440</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>송신 메시지:</string>
       </property>
      </widget>
      <widget class="QListWidget" name="receive_listWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>401</width>
         <height>271</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="user_name_label">
       <property name="geometry">
        <rect>
         <x>219</x>
         <y>10</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>사용자 이름</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QScrollBar" name="receive_ScrollBar">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>160</y>
         <width>16</width>
         <height>271</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="send_btn">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>470</y>
         <width>61</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>전송</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="signal_textEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>470</y>
         <width>341</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="room_name_lable">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>291</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
